test/CadlRanchProjects.Tests/azure-core-basic.cs

```diff
@@ -129,5 +129,23 @@ await foreach (Page<SecondItem> page in second.AsPages())
                 Assert.AreEqual("Madge", firstItem.Name);
             }
         });
+
+        [Test]
+        public Task Azure_Core_basic_confirmGetUserName() => Test(async (host) =>
+        {
+            User response = await new BasicClient(host, null).GetUserAsync(1);
+            var method = typeof(BasicClient).GetMethod("GetUserAsync");
+            var attribute = method.GetCustomAttributes<OperationAttribute>();
+            Assert.AreEqual(attribute.name, "getUser")
+        });
+
```

Comment:
We just want to check the name is correct. So
1. We don't need to call cadl-ranch server.
2. We just check the method exits.

Something like
```
[Test]
        public Task Azure_Core_basic_confirmGetUserName()
        {
            var method = typeof(BasicClient).GetMethod("GetUserAsync"); // Get method whose parameter is int
            Assert.IsNotNull(method)
        });
```